README.md

```diff
@@ -51,6 +51,10 @@ The output documentation should be located in the `docs/` folder, just open the
 - [CheckDomain](./src/general/CHECKDOMAIN.md)
 - [WhoAmI](./src/general/WHOAMI.md)
 
+### License
+
+- [License](./src/license/README.md)
```

Comment:
It's not very clear to have "License" and "Licenses" in this documentation
What is the point exactly? Why would I click on one or the other?
You should either group the docs in 1 file or explain clearly on this page why there are 2 different topics and what each one is about

Reply:
Done